src/js/components/TaskTable.js

```diff
@@ -39,9 +40,55 @@ class TaskTable extends React.Component {
   }
 
   getStatusValue(task) {
+    let taskHealth = this.getTaskHealth(task);
+    if (taskHealth === true) {
```

Comment:
I guess this makes it more precise to differentiate between e.g. `null` as opposed to boolean values.
I also don't like this function to read:
```js
if (taskHealth) {
  return 'Healthy';
}
if (taskHealth === false) {
  return 'Unhealthy';
}
...
```

Which to me would be more confusing than what is already there